app/experimenter/static/js/components/TimelinePopForm.js

```diff
@@ -94,6 +96,49 @@ class TimelinePopForm extends React.PureComponent {
     }
   }
 
+  displayRolloutPlaybookOrEnrollment() {
```

Comment:
I'd rather split this into two things like

    displayRolloutPlaybook
    displayEnrollmentDuration


and then inside each one just start with the bool check about whether it should show

Reply:
will do.